app/js/components/common/Header.jsx

```diff
@@ -64,9 +64,17 @@ export default class Header extends Component {
   }
 
   getCurrentLocation() {
-    this.fetchLocation('/v1/appui/session').then((response) => {
-      this.setState(() => ({ currentLocationTag: response.sessionLocation.name }))
-    });
+    let installed = this.isModuleInstalled('appui');
+    if (installed) {
+      this.fetchLocation('/v1/appui/session').then((response) => {
+        this.setState(() => ({ currentLocationTag: response.sessionLocation.name }))
+      });
+    } else {
+      // Use a location that exists with platform because of AOM-160
```

Comment:
Do help the reviewer not to jump around for that ticket and read to get a summary of it, wouldn't it be better to just expand it here?

Reply:
Yes, it's more intuitive to have a clear comment. I'm adding it in the following commit.